### ISSUE TITLE
Fetch rainbow-dock's IP address

### ISSUE THREAD

**Issue by asbjornenge** [asbjornenge/rainbow-dock-populator#4]
Fetch rainbow-dock's IP address so we don't need to use --apihost. Just specify rainbow-dock's container name in maybe a --apiname option.

**Comment by asbjornenge** [comment]
@simonqbs are thinking in case of linking here? So the apiname would be the linkname and the populator would pick the ip from env's?

**Comment by simonqbs** [comment]
Yes, that would work, but can't you just skip apiname and pick the ip from the linkname (if defined)?

**Comment by asbjornenge** [comment]
That's what I was thinking too... :+1: So if RAINBOW_DOCK_\* link env exists I'll use that ip. If not complain on a missing --apihost. It's a convention, but a good one :smile:
